changelog/4903.feature.rst

```diff
@@ -0,0 +1 @@
+Colorbar in `sunpy.map.GenericMap.peek` now has a unit label.
```

Comment:
The filename should be 4930, not 4903.

Reply:
Yes I shall make that change, my apologies.